homeassistant/components/led_ble/manifest.json

```diff
@@ -3,15 +3,17 @@
   "name": "LED BLE",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/ble_ble",
-  "requirements": ["led-ble==0.5.4"],
+  "requirements": ["led-ble==0.6.0", "flux_led==0.28.32"],
```

Comment:
Pinned it here since its using the same lib as flux_led under the hood for all the effects and color translation and they need to stay in sync